src/data/quot.lean

```diff
@@ -365,15 +365,19 @@ lemma nonempty_quotient_iff (s : setoid α) : nonempty (quotient s) ↔ nonempty
 
 /-! ### Truncation -/
 
+theorem true_equivalence : @equivalence α (λ _ _, true) :=
+⟨λ _, trivial, λ _ _ _, trivial, λ _ _ _ _ _, trivial⟩
+
+/-- Always-true relation as a `setoid`. -/
+def true_setoid : setoid α :=
+⟨_, true_equivalence⟩
```

Comment:
This already exists, it's `setoid.has_bot`

Reply:
It's not available in `data.quot`.

Reply:
Since this fell off the bors batch, I pushed a comment remarking that `⊤ : setoid α` is another spelling of the same thing.